storybook/src/foundation/design-tokens/border.docs.mdx

```diff
@@ -12,9 +12,9 @@ Elements that have a border, outline or underline use one of these widths.
 
 We have 4 border widths:
 
-| Size            | px  |  rem   | Token name            | Example                                                                                                             |
-| :-------------- | :-: | :----: | :-------------------- | ------------------------------------------------------------------------------------------------------------------- |
-| **Small**       |  1  | 0.0625 | `ams.border.width.sm` | <div className="ams-docs-token-preview--border" style={{ borderInlineStartWidth: 'var(--ams-border-width-sm)' }} /> |
-| **Medium**      |  2  | 0.125  | `ams.border.width.md` | <div className="ams-docs-token-preview--border" style={{ borderInlineStartWidth: 'var(--ams-border-width-md)' }} /> |
-| **Large**       |  3  | 0.1875 | `ams.border.width.lg` | <div className="ams-docs-token-preview--border" style={{ borderInlineStartWidth: 'var(--ams-border-width-lg)' }} /> |
-| **Extra large** |  4  |  0.25  | `ams.border.width.xl` | <div className="ams-docs-token-preview--border" style={{ borderInlineStartWidth: 'var(--ams-border-width-xl)' }} /> |
+| Token name            | px  |  rem   | Example                                                                                            |
+| :-------------------- | :-: | :----: | -------------------------------------------------------------------------------------------------- |
+| `ams.border.width.sm` |  1  | 0.0625 | <div className="ams-docs-token-example--border" style={{ borderInlineStartWidth: '0.0625rem' }} /> |
```

Comment:
Why the hardcoded value here, instead of a CSS var?

Reply:
I think they have always been like this. Updated now.